test/tools/yulInterpreter/EVMInstructionInterpreter.cpp

```diff
@@ -204,6 +205,7 @@ u256 EVMInstructionInterpreter::eval(
 	case Instruction::CALLDATASIZE:
 		return m_state.calldata.size();
 	case Instruction::CALLDATACOPY:
+		logTrace(_instruction, true, arg);
```

Comment:
`logTrace(instruction, true, arg)` means interpret instruction `I` whose args are `A...` and tell the tracer that this instruction may write to memory.

If the interpreter is configured to disable memory tracing, this information (whether an instruction writes to memory or not) is useful in deciding whether to append the instruction to execution trace or not.